packages/compass-schema-validation/src/stores/store.js

```diff
@@ -74,6 +81,9 @@ const configureStore = (options = {}) => {
     // set the initial value
     changeEditMode();
 
+    // set hadronReadOnly if readOnly value has cganged in Compass settings.
```

Comment:
```suggestion
    // set hadronReadOnly if readOnly value has changed in Compass settings.
```

(but also similarly here, it might be good to rename things, esp. since there is also nothing called …hadron… anymore related to readonly settings)